docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md

```diff
@@ -0,0 +1,107 @@
+
+- Press the *Win* key, search for *Command Prompt* and run it. There, type in `sfc /scannow` and press *Enter*.
+
+- Choose a different installation folder. It is possible that the current installation folder has some access restrictions. Also make sure you don't select an external drive, a virtual drive, etc.
+
+- Uninstall AdGuard VPN using our special [uninstall tool](../../installation#advanced) and then repeat the installation.
+
+- Start and re-register Microsoft Installer service. It requires some work.
+
+    1) Press *Win + R* and enter ***services.msc***.
+    2) Find in the list and double click on *Windows Installer*.
+    3) Hit *Start* button under *Service status* and hit *OK*. If the service status is **running**, you should click on *Stop* first and then hit *Start*. 
+    4) Press *Win + R*, type and enter ***msiexec /unregister*** and hit *Enter*.
+    5) Press *Win + R* again, type and enter ***msiexec /regserver*** and hit *Enter*
+
+- Acquire full permissions on the drive for installation. It is possible that the error 1603 occurs because you don’t have full permissions on the file location. It's also not as easy as some of the other solutions:
+
+    1) Open *File Explorer*, right click on the drive containing the installation location and select *Properties*.
+    2) Go to *Security* tab and click on *Edit*.
+    3) Single click on *SYSTEM* and ensure that the *Allow* box of every item in *Permissions for SYSTEM* is checked (if it is checkable). Do the same check for *Administrators*.
+    4) Click on *OK* to go back to *Properties* dialog. Then click on *Advanced*.
+    5) Click on *Change Permissions*.
+    6) On *Permissions* tab, double click on *Administrators*.
+    7) Select *This folder, subfolders and files* for *Applies to* field and tick all the available *Basic permissions*. After that hit *OK*.
+    8) Do the same operation above (from item 7) for *SYSTEM*.
+    9) Click on *OK* all the way out. Try installing AdGuard again.
+
+### Error 1618: Another installation is already in progress {#error-1618}
+
+This error occurs when there are several instances of AdGuard VPN installer launched at the same time. What to do if you get this error:
+
+- Reboot your PC and start the installer again. When you restart the computer, all ongoing processes will stop, including all copies of the installer.
+
+- Don't make multiple clicks on the installer even if doesn't start right away. Sometimes it may take a few seconds to display the installer UI.
+
+### Error 1638: Another version of this product is already installed {#error-1638}
+
+It's very likely that you've already installed AdGuard VPN before.
+
+- Check if AdGuard VPN is already installed on your computer. You can do it by pressing the *Win* key and typing in ***AdGuard VPN***.
```

Comment:
why ***?

Reply:
don't know, it was like this in the original text